weblate/trans/vcs.py

```diff
@@ -733,7 +733,7 @@ def push(self):
         on fork and creates pull request against original repository.
         """
         self.fork()
-        fork_branch = '{0}-weblate'.format(self.branch)
+        fork_branch = '{0}-weblate'.format(self.name)
```

Comment:
did you understand the scenario? for multiple subprojects, the fork branch name is created as master-weblate. For the first one, ok. But when resources are translated for the second one, it creates the same branch master-weblate and fails to push.
With the purposed fix, the branch name would be subproject1-weblate and subproject2-weblate, so it would not be the same branch name anymore.
Again, I don't know if it's the best solution at all, but do you have another idea to prevent this scenario error?
Just trying to help for future users, though. In our company we just edited the source code and it's working as we need. =)